### PR TITLE
MBUILD-148 Princess

### PR DESCRIPTION
**Pull request by jfduedu**
Added princess and other qol

### CODE REVIEW

**Comment by jfduedu** [comment]
I may have forgotten VS edits, will do

File: AdventureGame/Princess.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        PrintDisplay::custom_cout << "You run off into the forest as the pain of rejection fills your soul.\n";
+        PrintDisplay::custom_cout << "You weep and run as far as you can, never to be seen again.\n\n";
+        PrintDisplay::custom_cout << "BAD ENDING\n\n";
+        PrintDisplay::no_effect_flush();
+        return true;
+    }
+    else if (answer == 3)
+    {
+        PrintDisplay::custom_cout << "Princess:\n\t\"If you haven't got the guts to say anything,\n";
+        PrintDisplay::custom_cout << "\tthe get your pathetic ass out of my castle!\"\n\n";
```
Comment (NickPerillo4) [inline AdventureGame/Princess.cpp]: Spelling error "the" should be "then"

File: AdventureGame/AdventureGame.cpp [hunk 2]
```diff
@@ -148,20 +149,76 @@ int main()
...
+            else if (!princessIgnored)
+            {
+                endGame = dynamic_cast<Princess&>(*princess).rizzTime();
+            }
+
+            if (endGame == true) // End the game.
             {
-                PrintDisplay::custom_cout << "\nYOU DID NOT SURVIVE!\n";
+                PrintDisplay::custom_cout << "\nYour adventure has reached it's conclusion.\n";
+                PrintDisplay::custom_cout << "Thanks for playing!\n\n";
```
Comment (NickPerillo4) [inline AdventureGame/AdventureGame.cpp]: Should be "Thanks for playing Survival Kingdom!"

**Comment by NickPerillo4** [comment]
When the princess tries to push you and you dodge it. You fall into the moat which should be the end of the game but it asks what do you want to do

**Pull request merged**